isofit/core/geometry.py

```diff
@@ -53,20 +54,7 @@ def __init__(
         self.surface_elevation_km = None
         self.earth_sun_distance = None
         self.esd_factor = None
-
-        self.earth_sun_file = None
-        self.earth_sun_distance_path = os.path.join(
-            isofit.root, "data", "earth_sun_distance.txt"
-        )
-        try:
-            self.earth_sun_distance_reference = np.loadtxt(self.earth_sun_distance_path)
-        except FileNotFoundError:
-            logging.warning(
-                "Earth-sun-distance file not found on system. "
-                "Proceeding without might cause some inaccuracies down the line."
-            )
-            self.earth_sun_distance_reference = np.ones((366, 2))
-            self.earth_sun_distance_reference[:, 0] = np.arange(1, 367, 1)
```

Comment:
We need to port this over to the new version as well.

Reply:
Done